doc/source/raster_class.md

```diff
@@ -0,0 +1,337 @@
+---
+file_format: mystnb
+kernelspec:
+  name: geoutils
+---
+(raster-class)=
+
+# The georeferenced raster ({class}`~geoutils.Raster`)
+
+Below, a summary of the {class}`~geoutils.Raster` object and its methods.
+
+(raster-obj-def)=
+
+## Object definition and attributes
+
+A {class}`~geoutils.Raster` contains **four main attributes**:
+
+1. a {class}`numpy.ma.MaskedArray` as {attr}`~geoutils.Raster.data`, of either {class}`~numpy.integer` or {class}`~numpy.floating` {class}`~numpy.dtype`,
+2. an {class}`affine.Affine` as {attr}`~geoutils.Raster.transform`,
+3. a {class}`pyproj.crs.CRS` as {attr}`~geoutils.Raster.crs`, and
+4. a {class}`float` or {class}`int` as {attr}`~geoutils.Raster.nodata`.
+
+For more details on {class}`~geoutils.Raster` class composition, see {ref}`core-composition`.
+
+A {class}`~geoutils.Raster` also contains many derivative attributes, with naming generally consistent with that of a {class}`rasterio.io.DatasetReader`.
+
+A first category includes georeferencing attributes directly derived from {attr}`~geoutils.Raster.transform`, namely: {attr}`~geoutils.Raster.shape`,
```

Comment:
Maybe it would be nice to list these attributes in a table? Currently, it's a bit lost in the text.

Reply:
There's already a hyperlink to the API section just below